CHANGES.rst

```diff
@@ -601,6 +601,9 @@ astropy.coordinates
 - Negating a ``SphericalRepresentation`` object now changes the angular
   coordinates (by rotating 180º) instead of negating the distance. [#7988]
 
+- Creation of new frames now generally creates copies of frame attributes,
+  rather than inconsistently either copying or making references. [#8204]
```

Comment:
is there a number for an actual PR that made this change?

Reply:
I am not sure.  It was probably part of one of the representation updates, but probably a side effect of a couple different PRs.  Not clear to me which, though, hence my thinking it's better just to reference the point here for more context.